### ISSUE TITLE
Latest Release not downloading latest code 

### ISSUE THREAD

**Issue by veris-neerajdhiman** [aschn/drf-tracking#71]
- I have installed latest `Release 1.3.1` but it don't have latest code as you have in your master branch.

e.g., 
1) Earlier you have `_clean_data` as separate function but now you have moved it to class function and modified it too. Release 1.3.1 still have old code for this function  (I have checked source code of Release too)
2) The actual error I was getting is `TypeError: expected string or bytes-like object` in line No 137 `if SENSITIVE_DATA.search(key):`  
3) So when I checked installed Lib code and Master branch code, It was different. 
4) So please update Release code and also check if New code can fix my original error which is mentioned in 2nd point . 
5) Here data is a `list` of `dict()`
 ```json
[   {
        "actor_type": "tets",
        "meeting_with": "Self",
        "first_name": "qwert",
        "last_name": "asdfg",
        "logged_at": "1448113944"
    }
    ]

**Comment by triat** [comment]
Hello @veris-neerajdhiman, if you want to check if your error is fixed, you can also install the code from master directly with pip. 

Let us know if the issue is still there

**Comment by veris-neerajdhiman** [comment]
@triat  yes issue was still there so for now I have forked this repo and patched a host fix (foe loop) but you must verify the same at your end and apply permanent fix.

**Comment by triat** [comment]
@veris-neerajdhiman can I propose that you do a PR if you already did the fix ? and write tests maybe ? Much appreciated

**Comment by veris-neerajdhiman** [comment]
@triat  PR #73  created for this issue, Please check

**Comment by avelis** [comment]
@veris-neerajdhiman @triat 
I think the critical issue here that we are attempting force a `dict` copy on a post body that might not always translate to a dict. In that scenario we could consider using the polymorphic `copy.deepcopy` and make `_clean_data` be able to handle a list of dictionaries or just ignore cleaning a list structure.

Thoughts?

**Comment by veris-neerajdhiman** [comment]
@avelis  
 - I have used `copy.deepcopy()`  you can check in my commits. 
 - Secondly I think we should not ignore list structure as it may or may not include sensitive data.
